internal/webhook/external/server.go

```diff
@@ -32,8 +32,8 @@ func (s *server) Serve(ctx context.Context, l net.Listener) error {
 	logger := logging.LoggerFromContext(ctx)
 	mux := http.NewServeMux()
 
-	// TODO: Register handlers here
-	// mux.Handle("/", ...)
+	mux.Handle("POST /v1/github", githubHandler(s.client))
```

Comment:
Good for now and as soon as we have the `WebhookReceiver` CRD sorted out, `/` will be mapped to a different handler that muxes to a more specific one, like GitHub.